### PR TITLE
Document the none-column

### LINKED ISSUE #180: Question: Interpretation of output
I'm currently playing around with the example from the README and I am not sure if everything is as clear as it could be. Could you explain what `none` means in the output or point the user to the description?
```r
print(explanation$dt)
#>      none     lstat         rm       dis      indus
#> 1: 22.446 5.2632030 -1.2526613 0.2920444  4.5528644
#> 2: 22.446 0.1671903 -0.7088405 0.9689007  0.3786871
#> 3: 22.446 5.9888016  5.5450861 0.5660136 -1.4304350
#> 4: 22.446 8.2142203  0.7507569 0.1893368  1.8298305
#> 5: 22.446 0.5059890  5.6875106 0.8432240  2.2471152
#> 6: 22.446 1.9929674 -3.6001959 0.8601984  3.1510531
```

Or you could call it 'average prediction' and also show the difference, like [here](https://christophm.github.io/interpretable-ml-book/shapley.html)?

### PR DESCRIPTION
**Pull request by martinju**
Fixes #180 

Documents the non-column in the help file of `shapr::explain` + basic explanations commented-out in the code examples in README + vignette (and /inst/scripts/readme_example.R)

### CODE REVIEW

File: R/explanation.R [hunk 1]
```diff
@@ -50,6 +50,13 @@
 #' \code{j} and \code{k} both increased the value of the prediction, but that the effect of the k-th
 #' feature was larger than the j-th feature.
 #'
+#' The value in \code{dt[i, 1]} is the prediction value  not assigned to any of the features
+#' (\ifelse{html}{\eqn{\phi}\out{<sub>0</sub>}}{\eqn{\phi_0}})
+#' This is the same for all prediction and set by the user through the argument \code{prediction_zero}.
```
Comment (nikolase90) [inline R/explanation.R]:
```suggestion
#' It's equal for all observations and set by the user through the argument \code{prediction_zero}.
```

File: README.Rmd [hunk 2]
```diff
@@ -118,7 +118,10 @@ explanation <- explain(
   prediction_zero = p
 )
 
-# Printing the Shapley values for the test data
+# Printing the Shapley values for the test data.
```
Comment (nikolase90) [inline README.Rmd]: I think we should avoid commenting this much, and rather refer to `?shapr::explain`.

I.e. you could do 
```
# Printing the Shapley values for the test data. For more information about the interpretation of the values see ?shapr::explain.  
```

Comment (martinju) [reply README.Rmd]: Agree. Fixed with 5e8371ce6917e28ab0909336bc3a9dc054d8d8b8

File: R/explanation.R [hunk 3]
```diff
@@ -50,6 +50,13 @@
 #' \code{j} and \code{k} both increased the value of the prediction, but that the effect of the k-th
 #' feature was larger than the j-th feature.
 #'
+#' The value in \code{dt[i, 1]} is the prediction value not assigned to any of the features
+#' (\ifelse{html}{\eqn{\phi}\out{<sub>0</sub>}}{\eqn{\phi_0}})
+#' It's equal for all observations and set by the user through the argument \code{prediction_zero}.
+#' In theory this value should be the expected prediction without conditioning on any features.
+#' Typically we set this value to the mean of the response in our training data, but other choices
```
Comment (nikolase90) [inline R/explanation.R]: What is the difference between `mean of the response in out traning data` and `such as the mean of the predictions in the training data`?

Comment (martinju) [reply R/explanation.R]: mean of reponse = mean(y)
mean of predictions  = mean(pred(x))

might change to `such as the mean of the predictions of the training data`

Comment (nikolase90) [reply R/explanation.R]: Ah, of course.

Comment (nikolase90) [inline R/explanation.R]:
```suggestion
#' Typically we set this value equal to the mean of the response variable in our training data, but other choices
```

**Comment by nikolase90** [comment]
@martinju Made two small changes. Feel free to merge when tests are finished.

**Pull request merged**